kafkaesk/app.py

```diff
@@ -236,22 +250,35 @@ async def __run(self) -> None:
                         partition=record.partition,
                         group_id=self._subscription.group,
                     ).inc()
-                except UnhandledMessage:
-                    # how should we handle this? Right now, fail hard
+                except UnhandledMessage as err:
                     logger.warning(f"Could not process msg: {record}", exc_info=True)
-                    # We increase the error counter
                     CONSUMED_MESSAGES.labels(
                         stream_id=record.topic,
                         partition=record.partition,
                         error="UnhandledMessage",
                         group_id=self._subscription.group,
                     ).inc()
+                    try:
+                        await retry_policy(record=record, error=err)
```

Comment:
i think maybe using name `exception` instead of `error` could be clearer.

Reply:
Good ideal, I'll adjust that in the new PR